steps/user-add/spec.schema.json

```diff
@@ -0,0 +1,39 @@
+
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "npm": {
+      "type": "object",
+      "description": "An object of NPM configuration",
+      "properties": {
+        "token": {
+          "type": "string",
+          "description": "An NPM access token"
```

Comment:
If this isn't part of a connection, this should have `writeOnly: true`. This will let us show it as sensitive data and suggest using a `!Secret` in the future.